docs/source/user_guide/users.ipynb

```diff
@@ -0,0 +1,1602 @@
+{
```

Comment:
<blockquote>a <code>Tuple</code> of GeoArrow buffers</blockquote><p> </p><p>But... they are just plain cudf series and dataframes!</p><p> </p><p>I would propose we implement <a href="https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.from_file.html" target="_blank">`GeoSeries.from_file</a>with this function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I think I'd like to update `read_polygon_shapefile` to return a `GeoSeries`, as `from_file` implies many more file types to support. We could write `from_file` and check if it is a shapefile, then use our internal reader otherwise use `GeoPandas`, but performance would be poor for all other file types. What do you think? https://github.com/rapidsai/cuspatial/issues/715